alsa_settings/PTLH_HDA_AIOC.sh

```diff
@@ -0,0 +1,13 @@
+set -e
```

Comment:
Please update the commit message, should match the style of the repository and have a signed-off at the end.